drivers/bridge/bridge.go

```diff
@@ -782,7 +782,9 @@ func (d *driver) deleteNetwork(nid string) error {
 			logrus.Warn(err)
 		}
 		if link, err := d.nlh.LinkByName(ep.srcName); err == nil {
-			d.nlh.LinkDel(link)
+			if dellinkerr := d.nlh.LinkDel(link); dellinkerr != nil {
+				logrus.Warnf("Failed to delete interface (%s)'s link on endpoint (%s) delete: %v", ep.srcName, ep.id, dellinkerr)
```

Comment:
can you use the format with: `logrus.WithError(dellinkerr).Warnf`

Reply:
you maybe want this to be Errorf

Reply:
Thanks. i think using the same formats better